crates/wash-cli/tests/common/mod.rs

```diff
@@ -490,6 +491,15 @@ impl TestWashInstance {
             .output()
             .await
             .context("failed to stop host")?;
+        let install_dir = downloads_dir()?;
+        let lock_file_exists = tokio::fs::try_exists(install_dir.join(WASMCLOUD_PID_FILE)).await?;
+        if lock_file_exists {
```

Comment:
Nit: Might be worth adding a comment here. It took me a second to catch that you were deleting it as a cleanup step if it still existed